change/change-2416297f-f036-427e-bc7e-7693f953608c.json

```diff
@@ -0,0 +1,18 @@
+{
+  "changes": [
+    {
+      "type": "patch",
```

Comment:
I think both of these should be change type minor due to API changes

Reply:
I don't know why I thought these packages were v 0.x.x. Thanks for the review. I'm fixing them.